src/script/lua_api/l_env.cpp

```diff
@@ -743,6 +743,31 @@ int ModApiEnvMod::l_get_objects_inside_radius(lua_State *L)
 	return 1;
 }
 
+// get_objects_in_area(pos, minp, maxp)
+int ModApiEnvMod::l_get_objects_in_area(lua_State *L)
+{
+	GET_ENV_PTR;
+	ScriptApiBase *script = getScriptApiBase(L);
+	
+	v3f minp = read_v3f(L, 1);
+	v3f maxp = read_v3f(L, 2);
+	aabb3f box(minp, maxp);
```

Comment:
This part is missing the `* BS` multiply, which is necessary to convert the position params to node sized units.

Reply:
Thanks, fixed it.